src/test/suite/extension.test.ts

```diff
@@ -6,18 +6,84 @@
  */
 import * as assert from 'assert';
 import {expect} from 'chai';
+import path = require('path');
 import {SfCli} from '../../lib/sf-cli';
 import Sinon = require('sinon');
-import { _runAndDisplayPathless, _runAndDisplayDfa } from '../../extension';
+import { _runAndDisplayPathless, _runAndDisplayDfa, _clearDiagnostics } from '../../extension';
 import {messages} from '../../lib/messages';
 
 // You can import and use all API from the 'vscode' module
 // as well as import your extension to test it
 import * as vscode from 'vscode';
-// import * as myExtension from '../../extension';
 
 suite('Extension Test Suite', () => {
 	vscode.window.showInformationMessage('Start all tests.');
+	// Note: __dirname is used here because it's consistent across file systems.
+	const codeFixturesPath: string = path.resolve(__dirname, '..', '..', '..', 'code-fixtures');
+
+	suite('E2E', () => {
```

Comment:
Adding integration/E2E tests is technically part of another story, but the current structure of the extension required me to add at least one E2E test to get coverage numbers to where they needed to be.